skins/wmde19/src/components/layout/Content.vue

```diff
@@ -1,11 +1,33 @@
 <template>
-	<main>
-		<slot></slot>
-	</main>
+	<div class="container has-margin-top-112 has-margin-bottom-48">
+		<div class="columns">
+			<main class="column is-two-thirds has-background-white">
```

Comment:
not a fan of  `-white`. Does this come from bulma and can we change it?
Generally we shouldn't be concerned with the color itself, therefor it shouldn't be in the name of the css class.

Reply:
![image](https://user-images.githubusercontent.com/223326/55962087-8640d880-5c70-11e9-8d0f-3422bd82bc84.png)

It's a Bulma thing.